content/en/logs/processing/attributes_naming_convention.md

```diff
@@ -25,66 +25,86 @@ This can generate confusion. For instance, a client IP might have the following
 
 In this context, the number of created or provided attributes can lead to confusion and difficulty to configure or understand the environment. It is also cumbersome to know which attributes correspond to the the logs of interest and—for instance—correlating web proxy with web application logs would be difficult. Even if technologies define their respective logs attributes differently, a URL, client IP, or duration have universally consistent meanings.
 
-Standard Attributes have been designed to help your organization to define its own naming convention and to enforce it as much as possible across users and functional teams. The goal is to define a subset of attributes that would be the recipient of shared semantics that everyone agrees to use by convention.
+Standard Attributes have been designed to help your organization to define its own naming convention and onboard users across multiple teams on it. The goal is to define a subset of attributes that would be the recipient of shared semantics that everyone agrees to use by convention.
 
-### Setup standard attributes
 
-Log Integrations are natively relying on the [default provided set](#default-standard-attribute-list), but your organization can decide to extend or modify this list.
-The standard attribute table is available in Log Configuration pages, along with pipelines and other logs intake capabilities (metrics generation, archives, exclusion filters, etc.).
+## Aliasing, Faceting and Promoting Attributes
```

Comment:
I wonder if this section should not be set at the end, just before the `Default standard attribute list` as otherwise there is a lot of text and definitions to read before getting to the actual feature.